dev/src/dev/fe_helpers.cljs

```diff
@@ -13,3 +13,5 @@
 
 (defn current-query []
   (.-__MLv2_query js/window))
+
+
```

Comment:
Is this intended?

Reply:
Ah, no it's not. I added a new helper that didn't pan out and didn't clean it up all the way.